scripts/ci-entrypoint.sh

```diff
@@ -102,15 +102,16 @@ setup() {
     export CCM_COUNT="${CCM_COUNT:-1}"
     export WORKER_MACHINE_COUNT="${WORKER_MACHINE_COUNT:-2}"
     export EXP_CLUSTER_RESOURCE_SET="true"
-
     # this requires k8s 1.22+
+    if [[ -n "${K8S_FEATURE_GATES:-}" ]]; then
```

Comment:
do we need this for all clusters? I don't think it hurts anything

Reply:
IMO the answer is "yes" because "has a windows node" is not a function of a cluster, it is a potential cluster configuration state that may or may not exist at a given time. Explicitly disallowing your cluster to ever include key Windows functionality is a mistake (IMO).

Reply:
after 1.26 this flag is on by default anyways.

Reply:
cool, love me some non-controversy :)